Content.Shared/_RMC14/Medical/Pain/PainKnockOutSystem.cs

```diff
@@ -0,0 +1,60 @@
+using Content.Shared.Mobs;
+using Content.Shared.Mobs.Systems;
+using Content.Shared.Mobs.Components;
+using Content.Shared.StatusEffect;
+using Content.Shared.Mobs.Events;
+
+namespace Content.Shared._RMC14.Medical.Pain;
+
+public sealed class PainKnockOutSystem : EntitySystem
+{
+    [Dependency] private readonly MobStateSystem _mobState = default!;
+
+
+    [ValidatePrototypeId<StatusEffectPrototype>]
+    private readonly string _painKnockOutKey = "PainKnockOut";
```

Comment:
If this is not const you can make it ProtoId<string>
You might have to update usings to change this
```suggestion
    private readonly ProtoId<StatusEffectPrototype> _painKnockOutKey = "PainKnockOut";
```

Reply:
Not sure why it's not const, maybe previous versions of the code didn't compile, fixed